art-community-platform/backend/app/api/helpers/recommendation_helpers.py

```diff
@@ -40,5 +40,66 @@ def get_tags_of_favourites(user):
     return tags
 
 
+#get 8 most popular users in the system
+def get_popular_users():
+    try:
+        user_list = User.objects.all()
+
+    except:
+        return [False, "couldnt get the users from database"]
+
+    #This is the list that is sorted by number of followers in decreasing order
+    new_list = sorted(user_list,key=lambda user: len(user.followers),reverse=True)
+
+
+    if len(new_list)<8:
+        return new_list
+    else:
+        new_list[:8]
+
+    return new_list
```

Comment:
In this function, you sorted users according the number of followers in descending order and returned most popular 8 users since expected.  Very good!